jekyll/_cci2/contexts.md

```diff
@@ -7,104 +7,136 @@ categories: [configuring-jobs]
 
-**Note:** Bitbucket repositories do **not** provide an API that allows CircleCI contexts to be restricted, only GitHub projects include the ability to restrict contexts with security groups. Restricted Contexts are also **not** yet supported in private installations of CircleCI.
+To invoke a job that uses a restricted context, a user must be a member of one of the security groups for the context. If the user running the workflow does not have access to the context, the workflow will fail with the `Unauthorized` status.
 
 ### Restrict a Context to a Security Group or Groups
+{:.no_toc}
 
 You must be an organization administrator to complete the following task.
 
 1. Navigate to Organization Settings > Contexts page in the CircleCI app. The list of contexts appears. The default security group is `All members` and allows all users in the org to invoke jobs with that context.
-2. Click the Create Context button if you will use a new context or click the name of an existing context.
-3. Click the Add Group link. The Add Groups dialog box appears.
+2. Click the Create Context button if you wish to use a new context, or click the name of an existing context.
+3. Click the Add Security Group button to view the dialog box.
 4. Select GitHub teams or LDAP groups to add to the context and click the Add button. Use of the context is now limited to the selected groups.
 5. Click Add Environment Variables to add environment variables to the context if none exist and click the Add button. Use of the environment variables for this context is now limited to members of the security groups.
 6. Navigate to Organization Settings > Contexts in the CircleCI app. The security groups appear in the Security column for the context.
 
 Only members of the selected groups may now use the context in their workflows or add or remove environment variables for the context. 
 
 ### Approving Jobs that use Restricted Contexts
+{:.no_toc}
 
-Adding an approval job to a workflow allows a user to manually approve the use of a restricted context. 
-
-To restrict running of jobs that are downstream from an approval job, add a restricted context to those downstream jobs. 
+Adding an [approval job]({{ site.baseurl }}/2.0/configuration-reference/#type) to a workflow gives the option to require manual approval of the use of a restricted context. To restrict running of jobs that are downstream from an approval job, add a restricted context to those downstream jobs, as shown in the example below:
 
-For example, if you want the execution of job C and job D restricted to a security group, you need to add an approval job B before the jobs C and D to that use a context with a security group. That is, you can have four jobs in a workflow, job A can run unrestricted, the approval job B may be approved by any member, but the jobs C and D after the approval may only be executed by someone in the security group for the context used on jobs C and D. 
+```yaml
+workflows:
```

Comment:
As this is a partial example (not displaying the jobs that are referenced) I would maybe add a commend above the workflows that indicates other text is expected to exist (`# ...`)

Reply:
Oh yes good idea 👍